codeworld-base/src/Internal/CodeWorld.hs

```diff
@@ -22,7 +22,9 @@ module Internal.CodeWorld
     , animationOf
     , activityOf
     , groupActivityOf
+    , CW.unsafeGroupActivityOf
     , simulationOf
+    , CW.debugSimulationOf
```

Comment:
You want to write a wrapper around this, instead of exporting it directly.  The version of this from `codeworld-api` has the type `a -> (Double -> a -> a) -> (a -> CW.Picture) -> IO ()`.  But the export here should have the different type:

    ([Number] -> a,
     (a, Number) -> a,
     a -> Picture)
    -> Program

So you need a wrapper.  It should look almost identical to the `simulationOf` wrapper that already exists below.

Same comment applies to `debugInteractionOf`, once you get there.